tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = {py36}-django{111,20,21,22}
```

Comment:
Shouldn't this be `py35` to match the current production environment?

Reply:
changed.